sklearn/svm/src/libsvm/svm.cpp

```diff
@@ -3101,6 +3101,44 @@ const char *PREFIX(check_parameter)(const PREFIX(problem) *prob, const svm_param
 		free(count);
 	}
 
+	if(svm_type == C_SVC ||
+	   svm_type == EPSILON_SVR ||
+	   svm_type == NU_SVR ||
+	   svm_type == ONE_CLASS)
+	{
+	    PREFIX(problem) newprob;
```

Comment:
The surrounding code uses only tabs for indentation.

Reply:
Now the new code uses only tabs for identation, too